File: shared/src/commonMain/kotlin/app/UseCase.kt
package app

data class UseCase(
  val id: Identifier,
  val title: String,
//  val color: Color,
  val description: String = ""
) {
  enum class Identifier {
    PIXELSORT,
    NASA,
    NOTES,
    SETTINGS,
    FIBONACCI
  }
}

val appUseCases = listOf(
  UseCase(UseCase.Identifier.NASA, "Nasa", "Description"),
  UseCase(UseCase.Identifier.SETTINGS, "Settings", "Description"),
  UseCase(UseCase.Identifier.NOTES, "Notes", "Description"),
  UseCase(UseCase.Identifier.FIBONACCI, "Fibonacci", "Description"),
  UseCase(UseCase.Identifier.PIXELSORT, "Pixelsort", "Description")
)

File: ios/Playground/App/Models/UseCase.swift
//
//  UseCase.swift
//  Playground
//
//  Created by Fabian Heck on 04.11.19.
//  Copyright © 2019 appcom interactive GmbH. All rights reserved.
//

import Foundation
import UIKit

struct UseCase {
    
    var id: Identifier
    var title: String = ""
    var description: String = ""
    var imageName: String = ""
    var color: UIColor
    
    enum Identifier {
        case Pixelsort
        case Nasa
        case Notes
        case Locating
        case Settings
        case Fibonacci
        case Game
    }
}
